sunpy/tests/helpers.py

```diff
@@ -41,24 +43,33 @@
 else:
     SKIP_32 = True
 
-skip_windows = pytest.mark.skipif(platform.system() == 'Windows', reason="Windows.")
+skip_windows = pytest.mark.skipif(platform.system() == "Windows", reason="Windows.")
 skip_glymur = pytest.mark.skipif(SKIP_GLYMUR, reason="Glymur can not be imported.")
 skip_ana = pytest.mark.skipif(SKIP_ANA, reason="ANA is not available.")
-# Skip if the SunPy ASDF entry points are missing.
-asdf_entry_points = pytest.mark.skipif(not list(pkg_resources.iter_entry_points('asdf_extensions', 'sunpy')),
-                                       reason="No SunPy ASDF entry points.")
+if Version(python_version()) >= Version("3.10.0"):
+    asdf_entry_points = pytest.mark.skipif(
+        not entry_points().select(group="asdf_extensions", name="sunpy"),
+        reason="No SunPy ASDF entry points.",
+    )
+else:
+    asdf_entry_points = pytest.mark.skipif(
+        not all(
+            [
+                enter_point.name == "sunpy"
+                for enter_point in entry_points()["asdf_extensions"]
+            ]
+        ),
+        reason="No SunPy ASDF entry points.",
+    )
+new_hash_library = {}
```

Comment:
I think it makes sense to leave this next to  `get_hash_library_name()`?

Reply:
This is the only reference to `new_hash_library` so maybe just remove it? Probably left by mistake when the hash testing was moved to pytest-mpl.

Reply:
Removed.